packages/api/src/hooks/useTradingPlatformInvestorPasswordChange.ts

```diff
@@ -0,0 +1,22 @@
+import { useMemo } from 'react';
+import useRequest from '../useRequest';
+
+/** A custom hook that change the Trading Platform Investor Password. */
+const useTradingPlatformInvestorPasswordChange = () => {
+    const { data, ...rest } = useRequest('trading_platform_investor_password_change');
+
```

Comment:
@hamza-deriv don't we need to pass the payload to this hook? 
               payload: {
                   account_id:,
                   old_password,
                   new_password,
                   platform,
                }

Reply:
shouldn't it be passed from the component level like this:
```
const {mutate: tradingPlatformInvestorPasswordChange} = useTradingPlatformInvestorPasswordChange();
```

```
useTradingPlatformInvestorPasswordChange({
payload: {
    account_id: 'account_id',
    new_password: 'password',
    old_password: 'password',
    platform:'mt5',  //optional
},
});
```

Reply:
Actually at component level it'd be

```
const { mutate } = useTradingPlatformInvestorPasswordChange();
mutate({ account_id: 'account_id', new_password: 'password', old_password: 'password', platform:'mt5' });
```
I believe

Reply:
Yeah its depend on the usage of the hook from component level as discussed.
Actually forgot to update the description thats why it is confusing.
Now it is updated.
Thanks

Reply:
@hamza-deriv for mutation hook usually we return mutate function explicitly like the one thisyahlen did here https://github.com/binary-com/deriv-app/pull/9986/files. Its confuse us because you return data and the mutate is in `...rest` 😓

Reply:
Works both ways but its just consistency is what we wanna follow